src/Directory.Packages.props

```diff
@@ -13,10 +13,10 @@
     <PackageVersion Include="Microsoft.EntityFrameworkCore.SqlServer.NetTopologySuite" Version="6.0.8" />
     <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
     <PackageVersion Include="Microsoft.TypeScript.Compiler" Version="3.1.5" />
-    <PackageVersion Include="Microsoft.TypeScript.MSBuild" Version="4.9.4" />
+    <PackageVersion Include="Microsoft.TypeScript.MSBuild" Version="5.3.3" />
     <PackageVersion Include="Stateless" Version="5.12.0" />
     <PackageVersion Include="Sve.Blazor.InfiniteScroll" Version="1.0.0-alpha" />
-    <PackageVersion Include="Microsoft.AspNetCore.Components.Web" Version="6.0.13" />
+    <PackageVersion Include="Microsoft.AspNetCore.Components.Web" Version="8.0.2" />
     <PackageVersion Include="MudBlazor" Version="6.1.8" />
```

Comment:
A MudBlazort is frissíteni kell szerint, illetve ha már itt jársz légyszi frissíts fel minden .NEt 8-ra és a legfrissebb csomagokra

Reply:
Felfrissítettem minden projektet .Net8-ra és a legfrissebb csomagokra.

Reply:
köszi, viszont most látom, hogy egy kis jogosultsági gondom van ezen a repón, így nem fogom tudni még most bemergelni